content/en/tracing/enable/_index.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Enable trace collection in Datadog
+kind: Documentation
+aliases:
+  - /tracing/languages/
+  - /tracing/environments/
+  - /tracing/setup/environment
+  - /tracing/setup/first_class_dimensions
+  - /tracing/getting_further/first_class_dimensions/
+  - /agent/apm/
+further_reading:
+- link: "/agent/docker/apm"
+  tag: "Documentation"
+  text: "Docker APM setup"
+- link: "/integrations/amazon_ecs/#trace-collection"
+  tag: "Documentation"
+  text: "ECS EC2 APM setup"
+- link: "/integrations/ecs_fargate/#trace-collection"
+  tag: "Documentation"
+  text: "ECS Fargate APM setup"
+---
+
+To use APM, start by enabling trace collection in Datadog. You can do this in multiple different ways depending on your system setup: including using the [Datadog Agent locally](#Datadog-Agent), [on containers](#Containers), and [several other ways](#Other-ways-to collect-traces). For the full overview of all of the steps to set up APM, see the [APM overivew][1].
+
+## Datadog Agent
```

Comment:
the title doesn't match the description made in the overview of the page, i think if we want to stay consistent with the Agent terminology this could be called `Host Agent`